1208-projects/health_project.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Health related Project
+    
+    <Whay> --> In order to
+    <Who>  --> As a 
+    <What> --> I want to
+    (<How> --> One or more Scenarios, below Background)
+    
+    In order to do health research a project needs an REK approval
+    As a logged in User
+    I want to see the current REK approval of my project
+
+    In order to report correctly to HOD about health research project
+    As a Anonymus User
+    I want to see the Clinical Trial Phase for Drug studies on Landing Page
+
+    Rules:
+    * A project with a REK Approval is a Health Project
+    * A REK approval originate from REK, through a API call
+    * A REK approval containing a ProjectID, update that project 
+    * A REK approval lacking a ProjectID, originate a new Project
+    * Health Project need an updated REK Approval to change title, coordinating institution, start date, end data or participants of type PRO_MANAGER.
+
+    Questions:
+    * Do REK send us changed/updated approvals today?
+    * Do a REK approval have a field for our Project ID?
+    * How do we cleen up that REK is Project Owner on a lot of projects today?
+
+    # Below is <How> the <Whay>, <Who> and <What> is solved according to the rules
+    # Given bør kun brukes når det er ein bestemt sekvens av scenario - generelt kan Given utrykkes som ein del av Then -> DRY-code
+    # Så langt som mulig bør ikkje Scenario inneholde ord som click og button etc - det skaper vedlikeholdsutfordringer
+    # Test pyramiden oppfordrer til at omfanget av tester skal ligge mot "bunne" av stacken - ikkje på UI. Brudd på dette utfordrer CI, da build'n'deploy raskt vil overstige 10-15 min
+
```

Comment:
Background er helt nytt konsept for meg, men etter litt søking i confluence ser jeg at vi visstnok ble enige om å bruke det for snart ett år siden (https://unit.atlassian.net/wiki/spaces/NVAP/pages/729055292/2020-08-25+Standard+of+feature+descriptions)... 😬

Noe av det som står over her burde kanskje heller flyttes til en generell beskrivelse på conflunce i stedet for at det ligger litt ulike versjoner i diverse features?

Reply:
`Decision: Given’s should not refer to previous scenarios, but instead background should be stated explicitly` er stikk motsatt av det jeg har tenkt til nå i hvert fall...

Reply:
Det er sjelden at noko er rett eller galt. Given har sitt bruksområde, til dømes når du trenger å beskrive ein sekvens i detalj. Ulempen med Given er at du da ikkje kan skrive generiske sceanario, noko som gjer at du må skrive mange fleire og det resulterer i mykje meir (enkel) test kode - som krev mykje tid å vedlikeholde. 
Test pyramiden sei at du skal fokusere testene på modellen og ikkje GUI - mange GUI orienterte scenario drar test fokuset mot GUI, som i større grad endrer seg og det endrer seg raskere enn modellen.